mapeditor/mapview.cpp

```diff
@@ -270,6 +270,7 @@ void MapView::mousePressEvent(QMouseEvent *event)
 			if(event->button() == Qt::LeftButton)
 			{
 				auto * obj = sc->selectionObjectsView.selectObjectAt(tileStart.x, tileStart.y);
+				auto * obj2 = sc->selectionObjectsView.selectObjectAt(tileStart.x, tileStart.y, obj);
```

Comment:
consider better variable name than obj2 to reflect usage context

Reply:
Hm, but this is literally "object two" - second object selected by user. You can have overlapped objects on map